import_export_batches/controllers.py

```diff
@@ -993,7 +1064,7 @@ def create_batch_row_action_candidate(batch_description, batch_header_map, one_b
         # for this header_id (Duplicate entries in the same data set
         existing_batch_row_action_candidate_query = BatchRowActionCandidate.objects.all()
         existing_batch_row_action_candidate_query = existing_batch_row_action_candidate_query.filter(
-            batch_header_id=batch_description.batch_header_id, candidate_name=candidate_name,
+            batch_header_id=batch_description.batch_header_id, candidate_name=candidate_name, state_code=state_code,
```

Comment:
Thank you @mnneelam, I am going to approve this pull request, but wanted to request that every time you filter for state_code, please include "__iexact" like this:
state_code__iexact=state_code
You might also want to use "__iexact" with the candidate_name.